### ISSUE TITLE
Unable to resize while splitting to left or right

### ISSUE THREAD

**Issue by vicmns** [velitasali/GTKTitleBar#17]
When I split a window to the left or right, I'm unable to resize the window. This doesn't happen if the plugin is disabled.

This is a problem that is affecting all other plugin solutions for removing the title bar while on Fedora 30. So maybe it's is a change withing Gnome.

**Comment by velitasali** [comment]
I am running Fedora 30 on Wayland and previously was running Ubuntu 19.04 on Xorg, however, I never experienced this. Need more detail.

**Comment by vicmns** [comment]
Mmmm interesting, now that I'm on a fresh install of Fedora 30 running Wayland I cannot reproduce it. Previously I made a Fedora 29 Upgrade to Fedora 30 using the Nvidia proprietaries drivers and I was having the resize issue. Basically what happened is that once I made a split to right or left of the screen, the resize mouse pointer for left/right never appeared; so I was unable to resize.

Maybe is just an issue with XOrg + Nvidia drivers perse...
